canvas_account_admin_tools/templates/canvas_account_admin_tools/_no_tools_available.html

```diff
@@ -0,0 +1,20 @@
+{# todo: fix styles, or move a customizable template into icommons_common #}
```

Comment:
The styling on this works, and ultimately it should live in django-icommons-ui (typo in the code here) so it can be updated in just one place, like the default unauthorized message.